src/pages/groupComparison/GroupComparisonMutationsTabPlot.tsx

```diff
@@ -130,7 +132,11 @@ export default class GroupComparisonMutationsTabPlot extends React.Component<
                             plotLollipopTooltipCountInfo={
                                 plotLollipopTooltipCountInfo
                             }
-                            axisMode={this.axisMode}
+                            axisMode={
```

Comment:
again, just give the component an axisMode property and keep it agnostic to the url, which is really a concert of the page component

Reply:
fixed